proxy/coco/src/project/create/validation.rs

```diff
@@ -31,7 +28,11 @@ pub enum Error {
 
     /// An error occurred in `git2` that we could not handle.
     #[error(transparent)]
-    Git(#[from] git2::Error),
+    Git2(#[from] git2::Error),
+
+    /// An error occurred during a [`radicle_surf::git`] operation.
+    #[error(transparent)]
+    Git(#[from] git::error::Error),
```

Comment:
We don't need this error variant now, as far as I can tell. Could we remove it?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1278/commits/4500f4d7bd06e6d6fc39956767300c52e3e293c5